ui-components/src/jvmMain/kotlin/org/jetbrains/kotlinconf/ui/components/Gallery.kt

```diff
@@ -0,0 +1,48 @@
+package org.jetbrains.kotlinconf.ui.components
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Window
+import androidx.compose.ui.window.application
+import androidx.compose.ui.window.rememberWindowState
+import org.jetbrains.compose.reload.DevelopmentEntryPoint
+
+fun main() {
```

Comment:
If you do not want this code to be in your production sources, feel free to put them into the 'jvmDev' source set 👍 ! I think launching the code then requires the IDE plugin from Kostya or a cli command that I could provide!

Reply:
we also could have a Gradle task for this in the plugin :)

Reply:
Oh there is a task: 

```
gw :app:devRun -DclassName=my.pkg.ClassNameKt -DfunName=MyComposable
```

Reply:
I definitely prefer an easily clickable `main` function for the time being, just to keep things super simple, but we can revisit this later. A saved run config would make it fairly painless to launch the task above, for example.

I'm also still thinking about what entry point to provide exactly though, whether it should be a bunch of small ones for components, or if we should have a "gallery app" entry point where you can go through all of them.

Reply:
Absolutely fair!